src/module/actor/pokemon/document.js

```diff
@@ -354,27 +354,51 @@ class PTUPokemonActor extends PTUActor {
 
     // TODO: Implement rules for capability changing items
     _calcCapabilities() {
-        const capabilities = duplicate(this.species?.system?.capabilities ?? {});
-        if (!capabilities) return {};
+        const movementCapabilities = ["overland", "swim", "burrow", "sky", "levitate", "teleporter"]
+        const numericNonMovementCapabilities = ["highJump", "longJump", "power", "weightClass"]
+        const stringArrayCapabilities = ["naturewalk", "other"]
```

Comment:
Not a fan of pre-defining these both for homebrew reasons; as well as if we need to change it it'd have to be done on multiple pages, if something like this is to be used it should be added to `CONFIG.PTU.data`

Reply:
I agree with the putting them somewhere global.

The previous code already behaved with a hard coded list, I pulled it up. Tho one can argue that I "hard coded it harder" as before only the keywords "highJump", "longJump", "power", "weightClass", "other" were listed.

Reply:
@dylanpiera I centralized those in `Config.PTU.Capabilities` and found a single instance where the same hardcoded list is used applied it there. Also, there is no list of movementcapabilities anymore. It checks for not-being-in-the-other-lists.